### DISCUSSION TITLE
How to toggle cli flags/arguments in popup

### DISCUSSION THREAD

**Discussion by IvayloHrTonev** [NeogitOrg/neogit#1172] [Q&A] [answered]
Hi everyone,

When I open popup (for example NeogitPullPopup) I see a list of possible arguments
For example
```
│Arguments                                                                                                                                      │
│-f Fast-forward only (--ff-only)                                                                                                               │
│-r Rebase local commits (--rebase)                                                                                                             │
│-a Autostash (--autostash)                                                                                                                     │
│-t Fetch tags (--tags)
```
I don't know how and where to use them ?

**Comment by CKolkey** [comment, marked as answer]
Those are the cli flags available for git pull. You can read more about what they do in either `:h neogit` or `$ man git-pull`. You can toggle them using the characters to the left in the popup, like `-f`

**Comment by AddictArts** [comment]
Why show them then? If they are only for cli, then what is the value to neogit?

Other than helping with a `:! git pull -f` for example. Maybe I am missing something.

**Reply by AddictArts** [reply]
Oops, they will turn red when you type "-f" for example and if you check the $ history you can see them used. Thanks got my own answer.
